runtime/moonbase/src/asset_config.rs

```diff
@@ -159,7 +159,7 @@ impl pallet_asset_manager::AssetRegistrar<Runtime> for AssetRegistrar {
 			},
 		)
 		.get_dispatch_info()
-		.weight
+		.call_weight
```

Comment:
```suggestion
		.total_weight()
```

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/e1a01eeda369a6eb4e4d1b3cb4905916fd3c2ce6